### ISSUE TITLE
Loss function discussion in episode 3 breaks code checker

### ISSUE THREAD

**Issue by colinsauze** [carpentries-incubator/deep-learning-intro#191]
The explanation of the loss function and optimizer in the monitor the model section use #.... in the code example to indicate other parameters. Because this isn't valid code it breaks the code checker. 

The offending lines are:

~~~
model.compile(#...
              loss='mse',
              #...)
~~~

and

~~~
model.compile(optimizer='adam',
              loss='mse',
              #...)
~~~

I will enter a pull request with a proposed fix.

**Comment by dafnevk** [comment]
Fixed in #192
